const errorFunc = (data) => {
    throw new Error("[ERROR] " + data);
};
export default errorFunc;